packages/mask/src/plugins/EVM/storage/index.ts

```diff
@@ -0,0 +1,16 @@
+import type { ScopedStorage } from '@masknet/shared-base'
+import type { Web3Plugin } from '@masknet/plugin-infra'
+
+export const StorageDefaultValue = {
+    ensAddressBook: {} as Web3Plugin.EnsAddressBook,
+}
+
+let storage: ScopedStorage<typeof StorageDefaultValue> = null!
+
+export function setupStorage(_: typeof storage) {
```

Comment:
Haven't you forgotten to setup the storage? Learn more from `packages/plugins/Flow/src/UI/Dashboard/index.tsx`.

Reply:
No, Initialized in SNSAdaptor and dashboard.
packages/mask/src/plugins/EVM/UI/Dashboard/index.tsx
packages/mask/src/plugins/EVM/UI/SNSAdaptor/index.tsx